### ISSUE TITLE
won't connect to my open audible

### ISSUE THREAD

**Issue by magnificentdragonfly** [openaudible/openaudible#1172]
I get an error message with an error registering virtual device. When I follow the suggested link I deleted all devices associated with the app but it still won't log in. I also don't know how to check the console for more details.

**Comment by magnificentdragonfly** [comment]
Also I am on open audible 3.7.2

**Comment by gnarliprime** [comment]
I have this same issue. Tried deregistering all my audible devices, downgrading openaudible to 3.7 and installing on my linux box, all had the same results. Maybe this is the issue?

:java.lang.Exception: No browser cookies found. Log in again.

Log from console below with token and serial redacted

Mar 23, 2023 12:52:14 PM org.openaudible.audible.APIManager registerDevice
WARNING: register device failed:{
 "legacy": false,
 "serial": "<redacted>",
 "auth_mode": "TOKEN",
 "region": "US",
 "cookies": [],
 "token": "<redacted>",
 "username": ""
}:java.lang.Exception: No browser cookies found. Log in again.
	at org.openaudible.audible.APIManager.registerDevice(APIManager.java:224)
	at org.openaudible.audible.APIManager.takeLogin(APIManager.java:150)
	at org.openaudible.desktop.swt.manager.login.LoginInternal.handleMapLanding(LoginInternal.java:125)
	at org.openaudible.desktop.swt.manager.login.LoginInternal.takeURL(LoginInternal.java:163)
	at org.openaudible.desktop.swt.manager.login.LoginInternal.browserEvent(LoginInternal.java:190)
	at org.openaudible.desktop.swt.manager.browser.BrowserManager.browserEvent(BrowserManager.java:170)
	at org.openaudible.desktop.swt.manager.browser.swt.AudibleBrowser$1OurLocationListener.changed(AudibleBrowser.java:102)
	at org.eclipse.swt.browser.WebKit.webView_didCommitLoadForFrame(WebKit.java:971)
	at org.eclipse.swt.browser.WebKit.browserProc(WebKit.java:326)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend_bool(Native Method)
	at org.eclipse.swt.internal.cocoa.NSRunLoop.runMode(NSRunLoop.java:40)
	at org.eclipse.swt.widgets.Display.sleep(Display.java:5209)
	at org.openaudible.desktop.swt.gui.GUI.runEventLoop(GUI.java:292)
	at org.openaudible.desktop.swt.gui.GUI.run(GUI.java:391)
	at org.openaudible.desktop.swt.manager.AppLoader.<init>(AppLoader.java:133)
	at org.openaudible.desktop.swt.manager.AppLoader.main(AppLoader.java:146)
	at org.openaudible.desktop.Application.main(Application.java:48)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.exe4j.runtime.LauncherEngine.launch(LauncherEngine.java:84)
	at com.install4j.runtime.launcher.MacLauncher.main(MacLauncher.java:52)

**Comment by magnificentdragonfly** [comment]
I got it to work. I was having a problem downloading the update from 3.6.3 so I had uninstalled it and then installed the new 3.7.2 and that is when I had the problem with it connecting with audible. I just reinstalled the 3.6.3 version logged into audible with that version. Then reopened the 3.7.2 version and it all showed up as already logged in to the 3.7.2 version. I don't know why that worked but it did.
